utils/test/src/main/kotlin/Utils.kt

```diff
@@ -80,16 +80,25 @@ fun patchExpectedResult(
         .replaceIfNotNull("<REPLACE_URL_PROCESSED>", urlProcessed)
 }
 
+class ReplacementsBuilder {
+    val replacements = mutableMapOf<String, String>()
+
+    infix fun String.replaces(value: String) {
+        replacements[value] = this
+    }
+}
+
 fun patchExpectedResult2(
     expectedResultFile: File,
     definitionFile: File,
-    custom: Map<String, String> = emptyMap()
+    block: ReplacementsBuilder.() -> Unit = {}
```

Comment:
TBH, I liked the original *from*-*to*-syntax in the callers better, and I'd prefer to not introduce a helper class like `ReplacementsBuilder` for just this purpose. So my proposal would be to change this to

    block: MutableMap<String, String>.() -> Unit = {}

and also ... (see below)